frontend/src/router/index.ts

```diff
@@ -34,6 +35,10 @@ const router = createRouter({
       path: '/deleteknowledge',
       component: DeleteKnowledgeView
     },
+    {
+      path: '/deletelevel',
```

Comment:
```suggestion
      path: '/delete-level',
```

A path should be snake-case